x-pack/plugins/uptime/common/runtime_types/ping/synthetics.test.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  isRefResult,
+  isScreenshot,
+  isScreenshotRef,
+  isScreenshotImageBlob,
+  RefResult,
+  Screenshot,
+  ScreenshotImageBlob,
+  ScreenshotRefImageData,
+} from './synthetics';
+
+describe('synthetics runtime types', () => {
+  let refResult: RefResult;
+  let screenshot: Screenshot;
+  let screenshotImageBlob: ScreenshotImageBlob;
+  let screenshotRef: ScreenshotRefImageData;
+
+  beforeEach(() => {
```

Comment:
No reason in this particular block of tests - I find it can be a helpful pattern in cases where individual tests need to make small modifications to some larger structure. In cases like that it can be helpful to use a `beforeEach` in place ahead of that so when someone wants to mutate the objects above they wont' need to implement it.